app/templates/Form1.html.twig

```diff
@@ -509,17 +492,20 @@
 
                     $( '.payment-type-list input' ).change( function( evt ) {
                         store.dispatch( actions.newSelectPaymentTypeAction( evt.target.value ) );
-                        validators.amountValidator.validate().then( dispatchValidationResult );
                     } );
 
                     $( '#donation-submit1 button' ).click( function() {
                         store.dispatch( actions.newNextPageAction() );
                     } );
                 },
-
+                connectValidatorsToStore: function( store ) {
+                    var validators = this.newValidators();
+                    WMDE.ReduxValidation.createValidationMapper( store, [
+                            validators.amountValidator
+                    ] );
```

Comment:
If you just need this list of validators, why not just have a list to begin with, so you don't need to translate from map to list?